pom.xml

```diff
@@ -244,7 +244,7 @@
                 </goals>
               </execution>
             </executions>
-          </plugin>
+          </plugin>-->
```

Comment:
Heh, opps. Yep. That plugin doesn't work in codenvy, thought i reverted that before committing. I'll remove this change.